examples/minimal/packages/client-phaser/package.json

```diff
@@ -13,7 +13,7 @@
     "@improbable-eng/grpc-web": "^0.15.0",
     "@latticexyz/common": "link:../../../../packages/common",
     "@latticexyz/dev-tools": "link:../../../../packages/dev-tools",
-    "@latticexyz/phaserx": "link:../../../../packages/phaserx",
+    "@latticexyz/phaserx": "2.0.0-next.17",
```

Comment:
Curious which version is best to use here

Reply:
seems fine! surprised we were still publishing it with the rest